modules/phase_field/test/tests/mobility_derivative/AC_mobility_derivative_coupled_test.i

```diff
@@ -99,7 +99,7 @@
   scheme = 'bdf2'
 
   solve_type = 'NEWTON'
-  petsc_options_iname = 'pc_type'
```

Comment:
Does your PR do something to make these bad PETSc options actually generate an error?  It's pretty crazy that you can put complete garbage in there and the code doesn't do anything to let you know (which is obviously the root cause for all of these tests having bad options).